component-library/src/app/gallery/QA/michael/michael.component.ts

```diff
@@ -94,89 +94,15 @@ export class MichaelComponent implements OnInit {
     testFields: this.testerCheckboxConfig
   }
 
-  checkboxesConfigs: ICheckBoxComponentConfig[] = [
```

Comment:
Removing as no usage

Reply:
Triple check the board and with Keyur before pulling this. If it's good to get pulled out, put it on the component page.